plugins/woocommerce/includes/admin/helper/class-wc-woo-update-manager-plugin.php

```diff
@@ -94,30 +87,6 @@ function ( $product_data ) {
 		return (int) apply_filters( 'woo_update_manager_plugin_id', (int) $woo_update_manager_ids[0] );
```

Comment:
No need to change the product id to slug here, this can be implemented on the base PR, after mergign this.